lib/ex.coffee

```diff
@@ -199,9 +217,9 @@ class Ex
   substitute: (range, args) ->
     args = args.trimLeft()
     delim = args[0]
-    if /[a-z1-9\\"|]/i.test(delim)
+    if /[a-z]/i.test(delim)
       throw new CommandError(
-        "Regular expressions can't be delimited by alphanumeric characters, '\\', '\"' or '|'")
+        "Regular expressions can't be delimited by letters")
```

Comment:
Copypaste the lines from master and replace them? I think that's your best shot now.